src/components/FormElement.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {ForwardedRef, forwardRef, LegacyRef} from 'react';
+import {View, ViewProps} from 'react-native';
+import * as ComponentUtils from '@libs/ComponentUtils';
+
+type FormElementProps = ViewProps;
+function FormElement(props: FormElementProps, ref: ForwardedRef<View | HTMLFormElement | null>) {
+    return (
+        <View
+            role={ComponentUtils.ACCESSIBILITY_ROLE_FORM}
+            ref={ref as LegacyRef<View>}
```

Comment:
can we use the same type to avoid assertion?

Reply:
Also, could you clarify please why do you use LegacyRef here and why `ref` is defined as `ForwardedRef<View | HTMLFormElement | null>`?